lib/rubocop/config_loader.rb

```diff
@@ -220,6 +222,22 @@ def check_duplication(yaml_code, absolute_path)
         end
       end
 
+      def check_cop_config_value(hash, parent = nil)
+        hash.each do |key, value|
+          check_cop_config_value(value, key) if value.is_a?(Hash)
+
+          next unless %w[Enabled
+                         Safe
+                         SafeAutoCorrect
+                         AutoCorrect].include?(key) && value.class == String
```

Comment:
I think it is better to use `is_a?(String)`  instead of `== String`. Do you have any intentions?

Reply:
You're right.  It's just my habit, not deep intention. :pray: